packages/kbn-router-to-openapispec/src/generate_oas.ts

```diff
@@ -32,18 +32,19 @@ export interface GenerateOpenApiDocumentOptions {
   docsUrl?: string;
   tags?: string[];
   filters?: GenerateOpenApiDocumentOptionsFilters;
+  buildFlavour: string;
```

Comment:
```suggestion
  buildFlavour: 'serverless' | 'traditional';
```

Reply:
Got rid of buildFlavor as we won't be doing any offering based privileges for now. cc: @elena-shostak